charts/stable/minecraft-java/values.yaml

```diff
@@ -183,11 +199,57 @@ workload:
             # CF_OVERRIDES_SKIP_EXISTING
             # CF_PARALLEL_DOWNLOADS
             # CF_SET_LEVEL_FROM
+  mcbackup:
+    enabled: true
+    type: Deployment
+    strategy: RollingUpdate
+    replicas: 1
+    podSpec:
+      containers:
+        mcbackup:
+          primary: true
+          enabled: true
+          imageSelector: mcBackupImage
+          probes:
+            liveness:
+              enabled: false
+            readiness:
+              enabled: false
+            startup:
+              enabled: false
+          env:
+            BACKUP_METHOD: "tar"
+            DEST_DIR: "{{.Values.persistence.backups.mountPath }}"
+            SRC_DIR: "{{.Values.persistence.data.mountPath }}"
+            SERVER_PORT: "{{ .Values.service.main.ports.main.port }}"
+            RCON_HOST: "0.0.0.0"
+            RCON_PORT: "{{ .Values.service.rcon.ports.rcon.port }}"
+            RCON_PASSWORD: "{{ .Values.mcbackup.rcon_pass }}"
+            INITIAL_DELAY: "{{ .Values.mcbackup.initial_delay }}"
+            BACKUP_INTERVAL: "{{ .Values.mcbackup.backup_interval }}"
+            PRUNE_BACKUPS_DAYS: "{{ .Values.mcbackup.prune_backups_days }}"
+            PAUSE_IF_NO_PLAYERS: "{{ ternary true false .Values.mcbackup.pause_if_no_players }}"
```

Comment:
You don't need ternary to "print" a boolean if the `.Values.someinth` is already boolean
Its only needed if you want to print a string or something else
eg "{{ ternary "hasPlayers" "hasNot" .Values.mcbackup.pause_if_no_players }}"